packages/perseus/src/widgets/radio/__stories__/base-radio.stories.jsx

```diff
@@ -62,14 +82,115 @@ const defaultProps = {
     // an object with two keys: `checked` and `crossedOut`. Each contains
     // an array of boolean values, specifying the new checked and
     // crossed-out value of each choice.
-    onChange: ({checked, crossedOut}) => {},
+    onChange: action("changed"),
 
     // Whether this widget was the most recently used widget in this
     // Renderer. Determines whether we'll auto-scroll the page upon
     // entering review mode.
     isLastUsedWidget: false,
 };
 
-export const DefaultSettings = (args: StoryArgs): React.Node => {
-    return <BaseRadio {...defaultProps} />;
+export const Interactive = (args: StoryArgs): React.Node => {
+    const overwrittenProps = {...defaultProps, ...args};
+    return <BaseRadio {...overwrittenProps} />;
+};
+
+export const DefaultSingleSelect = (args: StoryArgs): React.Node => {
+    const overwrittenProps = {...defaultProps, multipleSelect: false};
+    return <BaseRadio {...overwrittenProps} />;
+};
+
+export const DefaultMultipleSelect = (args: StoryArgs): React.Node => {
+    const overwrittenProps = {...defaultProps, multipleSelect: true};
+    return <BaseRadio {...overwrittenProps} />;
+};
+
+export const DefaultMultipleSelectCountChoices = (
+    args: StoryArgs,
+): React.Node => {
+    const overwrittenProps = {
+        ...defaultProps,
+        multipleSelect: true,
+        numCorrect: 2,
+        countChoices: true,
+    };
+    return <BaseRadio {...overwrittenProps} />;
+};
+
+export const SingleSelected = (args: StoryArgs): React.Node => {
+    const choices = Array(4)
+        .fill(null)
+        .map((_, i) => generateChoice({content: `Choice ${i + 1}`}));
+    choices[1].checked = true;
+
+    const overwrittenProps = {...defaultProps, multipleSelect: false, choices};
+    return <BaseRadio {...overwrittenProps} />;
+};
+
+export const MultipleSelected = (args: StoryArgs): React.Node => {
+    const choices = Array(4)
+        .fill(null)
+        .map((_, i) => generateChoice({content: `Choice ${i + 1}`}));
+    choices[1].checked = true;
+    choices[2].checked = true;
+
+    const overwrittenProps = {...defaultProps, multipleSelect: true, choices};
+    return <BaseRadio {...overwrittenProps} />;
+};
+
+export const SingleKitchenSink = (args: StoryArgs): React.Node => {
+    const choices = Array(4)
+        .fill(null)
+        .map((_, i) => {
+            const choice = generateChoice({
+                content: `Choice ${i + 1}`,
+                rationale: "This is a neat rationale",
+                hasRationale: true,
+                showRationale: true,
+                correct: false,
+                showCorrectness: true,
+            });
+
+            return choice;
+        });
+    choices[1].checked = true;
+    choices[2].correct = true;
+
+    const overwrittenProps = {
+        ...defaultProps,
+        multipleSelect: false,
+        reviewModeRubric: {choices: (choices: $FlowFixMe)},
```

Comment:
I struggled with the reviewModeRubric type for awhile and finally gave up; this works and it's in a Story so I feel a little less bad about the `$FlowFixMe` (it's properly typed in the prod code).

Reply:
Looking more at `base-radio.jsx`, it really only seems like we're:
1. Looking to see if a rubric exists to determine if we're in review mode
2. Using the `choices[i].correct` to set styles

Content is the only required property, but I don't see how it's getting used. I get the impression that the rubric is referring to a legacy way of checking exercises. All this to say, this works without any noticeable changes in Storybook:

```
    const rubricChoices = choices.map(({correct}) => ({
        content: "",
        correct,
    }));

    const overwrittenProps = {
        ...defaultProps,
        multipleSelect: false,
        reviewModeRubric: {choices: rubricChoices},
        choices,
    };
    return <BaseRadio {...overwrittenProps} />;
```

Would this work for you?

Reply:
@kevinbarabash I went ahead and made that change. It's a little awkward, but I think refactoring `reviewModeRubric` is outside the scope of this PR, Flow's happy without `$FlowFixMe`, and Storybook looks the same either way AFAICT.